### ISSUE TITLE
Package my application failed with electron-packager

### ISSUE THREAD

**Issue by ozee31** [eliquious/electronify-server#7]
I try to package my application with electron-packager.

I have not errors but when run my exe nothing is happening.

Have you ever packager an application ?

Thk

**Comment by eliquious** [comment]
I have not tried `electron-packager` with it. If it runs normally, do you think it could be an issue with the packager config?

**Comment by ozee31** [comment]
When run command `electron .` whith my application or your full exemple it's run.

But when try to package my application or your full exemple '.exe" does'nt work.

With the basic exemple it's work.

How do you package your applications?

**Comment by eliquious** [comment]
I haven't tried packaging before now. I packaged one of the examples successfully and it ran but the window did not show. I can try to get an example running tonight if not earlier.

**Comment by Cir0X** [comment]
Any update on this?

**Comment by eliquious** [comment]
I didn't have any immediate luck with `electron-packager` a couple months ago. I haven't tried it with [electron-builder](https://github.com/electron-userland/electron-builder) yet but I plan to. When I initially looked at the issue it seemed like the child process was never starting and the UI was never being displayed as a result. 

It is on my short list for the holidays so I plan to knock it out soon unless you want to take a swing at it first.

**Comment by Cir0X** [comment]
I've successfully packaged my app for windows with `electron-packager`.
It seems that the packed application puts my server stuff into `resources/app`, so my electronify arguments looks like that:

```javascript
{
    command: 'node',
    args: ['dist/app.js'],
    options: {
        cwd: 'resources/app/server'
    },
    url: 'http://localhost:8080',
...
}
```

Furthermore the `child-process-promise` module, which are you using to spawn the server process, uses in version `2.2.0` `cross-spawn` which supports windows.
Therefore I've forked your project and updated the `child-process-promise`.
I can create a merge request if you want.
